.jenkins/Jenkinsfile

```diff
@@ -123,7 +173,15 @@ def win2016ReleaseCrossPlatform() {
             def oetoolsWin = docker.build("oetools-wincp", "-f .jenkins/Dockerfile .")
             oetoolsWin.inside {
                 timeout(15) {
-                    sh './scripts/test-build-config -p SGX1FLC -b Release --compiler=clang-7'
+                    dir('build') {
+                        withEnv(["CC=clang-7","CXX=clang++-7","OE_SIMULATION=1"]) {
+                            sh """
+                            cmake .. -DCMAKE_BUILD_TYPE=Release
+                            make
+                            ctest --output-on-failure
```

Comment:
Same comments as before -- you don't really need the ctest. The OE_SIMULATION flag does not matter here as we only need to build the ELF enclaves on Linux. It was overkill to use the test-build-config script for this step but was probably used for convenience.